### ISSUE TITLE
This seems to be a pre-built javascript file

### ISSUE THREAD

**Issue by berdof** [centrifugal/centrifuge-js#10]
I am using reactJS with webpack. And trying to import your library have the following warning 

`./~/centrifuge/centrifuge.js
Critical dependencies:
1:441-448 This seems to be a pre-built javascript file. Though this is possible, it's not recommended. Try to require the original source to get better results.
@ ./~/centrifuge/centrifuge.js 1:441-448`

This how I import the library  
`import centrifuge from 'centrifuge';`

Any thoughts how can it be used in es6 style way?

**Comment by FZambia** [comment]
@berdof hello!

This is something new to me too. Will try to find information what this even mean:)

**Comment by FZambia** [comment]
What I can say at moment - is it possible to import centrifuge from src directory somehow (which contains a source file not compiled with browserify)? 

I found many similar issues on Github in other projects - for example this one (https://github.com/justinwoo/elm-simple-loader/issues/1). Could this help?

**Comment by berdof** [comment]
@FZambia in this case I need to write something like this 
`import Centrifuge from '../node_modules/centrifuge/src/centrifuge.js';`. 
I did it and warning  dissapeared.

**Comment by FZambia** [comment]
Does this solution ok for you?

I've never used webpack and es6 imports – so don't really know can I make something from my side to make it work in a more correct way. Or just leave it as it is now?

**Comment by berdof** [comment]
@FZambia This works fine. But then we have even bigger issue with sockjs. It stays unclear for me how to use them both in es6 way. Because seems like sock defines globally (

**Comment by FZambia** [comment]
Could you make the simplest repo to reproduce this? So I can also try to experiment just cloning it?

**Comment by FZambia** [comment]
Reproduced it myself, so looks like I should do a way to explicitly provide SockJS object:

```
import Centrifuge from 'centrifuge'
import SockJS from 'sockjs-client'

var cent = new Centrifuge({
    "url": "1",
    "user": "2",
    "token": "3",
    "timestamp": "4",
    "debug": true,
    "sockJS": SockJS
});

cent.connect();
```

**Comment by FZambia** [comment]
Speaking about warning issue. Looks like setting "main" to "src/centrifuge.js" in `package.json` file solves it. But I am not sure about this solution backwards compatibility yet

**Comment by FZambia** [comment]
Just released version 1.3.7 with warning and SockJS dependency fixes - see [example in changelog](https://github.com/centrifugal/centrifuge-js/blob/master/CHANGELOG.md#137)
